Source/WebCore/testing/Internals.cpp

```diff
@@ -1334,7 +1335,7 @@ bool Internals::animationWithIdExists(const String& id) const
 
 unsigned Internals::numberOfActiveAnimations() const
 {
-    return frame()->document()->timeline().numberOfActiveAnimationsForTesting();
+    return frame() ? frame()->document()->timeline().numberOfActiveAnimationsForTesting() : 0;
```

Comment:
I’d think this would need to be in a `RefPtr` or call `protectedFrame()` in our new safer C++ model.

Reply:
Done.